src/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@
 if(NOT TOKEN_AUTH_ONLY)
 endif()
 
-find_package(Qt5 5.6 COMPONENTS Core Network Xml Concurrent REQUIRED)
+find_package(Qt5 5.6 COMPONENTS Core Network Xml Concurrent Gui REQUIRED)
```

Comment:
Hm I'm not sure about this one :-/

Reply:
Me neither, I just use it for QColor, could implement it manually but...
In general I'm not 100% happy how the utility code is handled here.

Reply:
We don't want libsync to depend on QtGui. I think this was a problem before. Possibly `Theme` should be split into a sync and a gui part, ideally in a way that makes ownbrander continue to work. 

You could look at doing that, but I guess that avoiding QColor here will be easier.

Reply:
I removed the dep to qtgui, and computate the lightness myself